secp256k1-sys/depend/secp256k1/src/asm/field_10x26_arm.s

```diff
@@ -27,16 +27,16 @@ Note:
 	.set field_not_M, 0xfc000000	@ ~M = ~0x3ffffff
 
 	.align	2
-	.global secp256k1_fe_mul_inner
-	.type	secp256k1_fe_mul_inner, %function
+	.global rustsecp256k1_v0_1_0_fe_mul_inner
```

Comment:
is this function linked as a public function?

Reply:
No idea, it's changed by autoreplace.